projects/kit/directives/connected/connected.style.less

```diff
@@ -0,0 +1,40 @@
+@import '@taiga-ui/core/styles/taiga-ui-local';
+
+[tuiConnected] {
+    --tui-icon-height: 2rem;
+    --tui-icon-width: 2rem;
+    --tui-left-offset: 1rem;
+    --tui-vertical-offset: calc((100% + var(--tui-icon-height)) / 2 + 4px);
+    --tui-connected-height: calc(100% - var(--tui-vertical-offset));
```

Comment:
We try to use `--tui-` prefix for public API vars and `--t-` for internal that are not meant to be provided externally.